interfaces/hr-time.idl

```diff
@@ -1,13 +1,14 @@
+
```

Comment:
Seems like we need to strip leading newlines and not just extra trailing ones. https://w3c.github.io/hr-time/ has an initial space in the first `<pre>` in the HTML sent, so that's where it came from.

Reply:
Done - See https://github.com/GoogleChromeLabs/webidl-diff/commit/704576b38fa8431aa5870d45d7d3444202f639b5